blueprint/index.md

```diff
@@ -0,0 +1,254 @@
+
+## Prerequisites
+
+### Specialized knowledge
+
+* Administrator-level knowledge of Genesys Cloud
+
+### Genesys Cloud account
+
+* A Genesys Cloud CX 1 license. For more information, see [Genesys Cloud Pricing](https://www.genesys.com/pricing "Opens the Genesys Cloud pricing article").
+* The Master Admin role in Genesys Cloud. For more information, see [Roles and permissions overview](https://help.mypurecloud.com/?p=24360 "Opens the Roles and permissions overview article") in the Genesys Cloud Resource Center.
+
+## Configure Genesys Cloud
+
+### Create custom roles for use with Genesys Cloud OAuth clients
+
+Create a custom role for use with a Genesys Cloud OAuth client with the following permissions.
+:::primary
+  **Note:** Custom role 2 is only required if you would still like GC users to make Communicate/PBX calls to other GC users.
+  :::
+
+| Roles           | Permissions | Role Name |
+|-----------------|-------------------------|---------|
+| Custom role 1 | **Conversation** > **Communication** > **Disconnect**, **Conversation** > **ExternalTag** > **Edit**  | Terminate Conversation Public API |
+| Custom role 2 | **Analytics** > **Conversation Detail** > **View** | Get Conversation Details Public API |
+
+To create a custom role in Genesys Cloud:
+
+1. Navigate to **Admin** > **Roles/Permissions** and click **Add Role**.
+
+   ![Add a custom role](images/createRole.png "Add a custom role")
+
+2. Enter the **Name** for your custom role.
+
+    ![Name the custom role](images/nameCustomRole.png "Name the custom role")
+
+3. Search and select the required permission for each of the custom role.
+   ![Add permissions to the custom role](images/assignPermissionToCustomRole.png "Add permissions to the custom role")
+4. Click **Save** to assign the appropriate permission to your custom role.
+
+:::primary
+  **Note:** Assign this custom role to your user before creating the Genesys Cloud OAuth client.
+  :::
+
+### Create an OAuth client for use with a Genesys Cloud data action integration
+
+To enable a Genesys Cloud data action to make public API requests on behalf of your Genesys Cloud organization, use an OAuth client to configure authentication with Genesys Cloud.
+
+Create an OAuth client for use with the data action integration with the following custom roles.
+:::primary
+  **Note:** OAuth Client 2 is only required if you would still like GC users to make Communicate/PBX calls to other GC users.
+  :::
+
+
+| OAuth Client   | Custom role | OAuth Client Name |
+|----------------|-------------------------------|-------|
+| OAuth Client 1 | Terminate Conversation Public API | Terminate Conversation Public API |
+| OAuth Client 2 | Get Conversation Details Public API | Get Conversation Details Public API |
+
+
+To create an OAuth Client in Genesys Cloud:
+
+1. Navigate to **Admin** > **Integrations** > **OAuth** and click **Add Client**.
```

Comment:
@AgnesCorpuz When you start looking at converting this to a blueprint, we should review the Genesys Cloud remote modules and the CX Accelerator that installs Data actions and see if these items need to be added.